packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -38,7 +39,8 @@ class DatadogTracer {
       [formats.TEXT_MAP]: new TextMapPropagator(config),
       [formats.HTTP_HEADERS]: new HttpPropagator(config),
       [formats.BINARY]: new BinaryPropagator(config),
-      [formats.LOG]: new LogPropagator(config)
+      [formats.LOG]: new LogPropagator(config),
+      text_map_dsm: new DSMTextMapPropagator(config)
```

Comment:
Should this not have its own formats key?

Reply:
yes it should, fixing